src/relax/transform/lift_transform_params.cc

```diff
@@ -494,22 +700,71 @@ class ConsumeBundledParams : public ExprMutator {
   std::unordered_map<int, Expr> param_remap_;
 };
 
+std::vector<std::pair<GlobalVar, Function>> GetTargetFunctions(
+    const IRModule& mod, const Array<String>& target_function_names) {
+  std::vector<std::pair<GlobalVar, Function>> target_functions;
+  if (target_function_names.size()) {
```

Comment:
With accepting a `Variant<Bool, Array<String>>` for the shared transforms, that argument would then be forwarded directly to the `GetTargetFunctions` utility.

```c++
if(auto function_names = shared_transforms.as<Array<String>>()) {
    for(const auto& name: function_names.value()) {
        ...
     }
} else {
    ...
}
```